### ISSUE TITLE
Result after training

### ISSUE THREAD

**Issue by Andredenise** [devJWSong/gpt2-chatbot-pytorch#9]
Do you also get a run folder after training with a meta, index, data file etc? So that I can use it in the interactive_conditional_samples.py from the gpt reposotory?

**Comment by devjwsong** [comment]
Hi, thank you for your opinion.

I don't understand what "a run folder after training with a meta, index, data file" means, but unfortunately the only thing saved as a checkpoint is model parameters, optimizer & scheduler currently.
If you clarify what additional information should be stored, I'm happy to consider and update it.

Thank you.

**Comment by Andredenise** [comment]
I will clarify myself. I once trained the gpt-fine-tuning and then I got a folder that looked like this.

checkpoint
counter
encoder
events.out.tfevents.1641649983
hparams
model-2000.data-00000-of-00001
model-2000.index
model-2000.meta
vocab.bpe

I need those files so that I can use the model within another project that I can interact with. My question to you is of after training you get the same kind of files?

**Comment by devjwsong** [comment]
I see. I think those files are supported in the scripts you used to train the GPT-2 before.
In this project, only model checkpoint, optimizer, and scheduler are saved, unfortunately.

I wonder what *.meta, *.index, *.data, counter, and encoder files are for specifically.
If you let me know, I will find a way to include them and update.

Thank you.
